selfdrive/car/gm/values.py

```diff
@@ -84,6 +85,9 @@ class GMCarInfo(CarInfo):
   CAR.BUICK_REGAL: GMCarInfo("Buick Regal Essence 2018"),
   CAR.ESCALADE_ESV: GMCarInfo("Cadillac Escalade ESV 2016", "Adaptive Cruise Control (ACC) & LKAS"),
   CAR.BOLT_EUV: GMCarInfo("Chevrolet Bolt EUV 2022-23", "Premier/Premier Redline Trim", video_link="https://youtu.be/xvwzGMUA210", footnotes=[], harness=Harness.gm),
+  # TEMP NOTE: ACC is not standard, only optional on LTZ and High Country Trims if you buy "Safety Package II"
+  # https://cdn.dealereprocess.org/cdn/brochures/chevrolet/2020-silverado1500.pdf
```

Comment:
what's the note here for? Safety Package II seems accurate

Reply:
hmm, the 2022 Silverado ACC package is different. perhaps we should just use "Adaptive Cruise Control" instead. @JasonJShuler do you expect the 2022 silverado will be the same platform or require a new port?

Reply:
@JasonJShuler mentioned that the Suburban/Tahoe went Global B in 2022, but not the Silverado. So 2023 likely might need to be a new platform

Reply:
Actually I think we support the 2022 as well. From @JasonJShuler: `Yes - If the 2023 is out I think it's still included... the Suburban / Tahoe went Global B in 2022, but no the silverado`

Reply:
2022 Silverado ACC packages: https://www.chevrolet.com/content/dam/chevrolet/na/us/english/index/shopping-tools/download-catalog/09-pdf/2022-chevrolet-new-silverado-1500-ebrochure.pdf